content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -15,6 +16,9 @@ This page shows how to enable and configure encryption of secret data at rest.
 
 * etcd v3.0 or later is required
 
+* To encrypt a custom resource, your cluster must be running Kubernetes v1.26 or newer.
```

Comment:
Somewhere (not exactly sure where), we need to address the fact that it does not matter if a CRD doesn't exist, you can still ask for it to be encrypted at rest.  If in the future the CRD gets created, it will be encrypted.  If it already existed before that point, a storage migration would be required to get all the CRs into an encrypted state (just like any other built-in resource).

Reply:
Possibly https://kubernetes.io/docs/concepts/security/security-checklist/ - and link to that point from elsewhere.

Alternative that is more work: make a page like https://kubernetes.io/docs/concepts/security/secrets-good-practices/ but for good practice around extending Kubernetes, and list it there. That new page could also be a place to, for example, mention that validating webhooks should go in _before_ CRDs.

Reply:
@enj PTAL https://github.com/kubernetes/website/pull/38109/files#diff-3e5cacff10969ee612fa982265f6d813bde8719ed49a18950e4380ae3de6b9c7R72-R76